ha/pcswrap/tests/status-xml-plain-resources.xml

```diff
@@ -80,10 +80,6 @@
     <tickets>
     </tickets>
     <bans>
-<!--
-# TODO: Replace with non-offensive terms when
-# https://bugs.clusterlabs.org/show_bug.cgi?id=5437 is resolved
```

Comment:
I hope we have a valid reason to remove this.

Reply:
i think it is needed i have not seed in cortx1.0 
but had discussion with @ArchanaLimaye 
Added this back